ocf_datapipes/load/pv/pv.py

```diff
@@ -294,6 +294,19 @@ def _load_pv_metadata(filename: str) -> pd.DataFrame:
     if "array_tilt_degrees" in pv_metadata.columns:
         pv_metadata["tilt"] = pv_metadata["array_tilt_degrees"]
 
+    # Need to change orientation to a number if a string (i.e. SE) that PVOutput.org uses by default
+    mapping = {
+        "S": 180.0,
+        "SE": 135.0,
+        "SW": 225.0,
+        "E": 90.0,
+        "W": 270.0,
+        "N": 0.0,
+        "NE": 45.0,
+        "NW": 315.0,
+        "EW": np.nan,
```

Comment:
@dantravers Do you happen to know what east-West means for orientation? Seems like it shouldn't be possible. This is from PVoutput.org data, we need to map the string orientations to actual numbers.

Reply:
I am not sure.  Maybe it's on their website, but my first guess is that it means panels on both the east and west sides of a roof.

Reply:
Okay, yeah, on the website https://pvoutput.org/help/data_services.html?highlight=orientation#api-get-system-orientation-codes it just lists it as one of the valid codes, and its definitions list https://pvoutput.org/help/terms_and_definitions.html?highlight=orientation just the main directions. Seems like rather than give a very wrong orientation, we just make it NaN and can filter it out later? This is only currently used by the pvlib normalization.